lib/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'dify-bin' {
+  import BManager from 'bin-manager';
+
+  export default function dify(images: string[], callback: Function): typeof BManager;
```

Comment:
It would be more clear if we change `images: string[]` to `arguments: string[]`.

Reply:
Noted, I will change that now :+1: